js/controller/ContentController.js

```diff
@@ -147,6 +151,8 @@ app.controller('ContentController', function (Publisher, FeedResource, ItemResou
         return $route.current.$$route.type === FEED_TYPE.FEED;
     };
 
+    this.oldestFirst = getOrdering();
```

Comment:
This variable is used in the template to tell if we want normal or reverse order. It needs to be a class variable so that the template can read it. (Previously, it was only defined inside the `this.autoPage` function.